openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -2,15 +2,29 @@
 Tagging API Views
 """
 from django.http import Http404
-from rest_framework.viewsets import ModelViewSet
+from rest_framework import status
+from rest_framework.viewsets import ModelViewSet, ReadOnlyModelViewSet
+from rest_framework.response import Response
+from rest_framework.pagination import PageNumberPagination
```

Comment:
We don't need to add pagination here anymore, this was added by https://github.com/openedx/openedx-learning/pull/69.

(If when we're building the frontend, we find that we need to change the pagination defaults, we'll handle that as a separate PR.)